homeassistant/components/unifiprotect/event.py

```diff
@@ -79,8 +100,19 @@ def _async_update_device_from_protect(self, device: ProtectDeviceType) -> None:
             and event.type is EventType.NFC_CARD_SCANNED
         ):
             event_data = {ATTR_EVENT_ID: event.id}
```

Comment:
The dict could be created all at once here instead of setting each key / value below